src/index.css

```diff
@@ -1,7 +1,3 @@
-html {
```

Comment:
Have you tried modify the html's font-size customly in the chart's component? Also if the chart accepts custom units we can try to give it rems